parser/parser.y

```diff
@@ -2424,7 +2424,7 @@ ReservedKeyword:
 "ADD" | "ALL" | "ALTER" | "ANALYZE" | "AND" | "AS" | "ASC" | "BETWEEN" | "BIGINT"
 | "BINARY" | "BLOB" | "BOTH" | "BY" | "CASCADE" | "CASE" | "CHANGE" | "CHARACTER" | "CHECK" | "COLLATE"
 | "COLUMN" | "CONSTRAINT" | "CONVERT" | "CREATE" | "CROSS" | "CURRENT_DATE" | "CURRENT_TIME"
-| "CURRENT_TIMESTAMP" | "CURRENT_USER" | "DATABASE" | "DATABASES" | "DAY_HOUR" | "DAY_MICROSECOND"
+| "CURRENT_TIMESTAMP" | "CURRENT_USER" | "DATABASE" | "DATE"  | "DATABASES" | "DAY_HOUR" | "DAY_MICROSECOND"
```

Comment:
`DATE` is not reserved words. if it's reserved, it should be `DATE[R]` in the table. @dreamquster